hod/local.py

```diff
@@ -76,7 +79,7 @@ def known_cluster_labels():
     """
     Return list of known cluster labels.
     """
-    path = cluster_config_dir()
+    path = cluster_info_dir()
     if os.path.exists(path):
```

Comment:
`OSError` has a field `.errno` (or error?) and you can check vs `errno.ENOENT`. But find, it's not so bad compared to fancylogger and not having zipped eggs.